r2c.bandit:
  violations:
    20abcea1b24eb8ff1e6aed1c33f47fb2:
      check_id: B602
      message: subprocess call with shell=True identified, security issue.
      path: bento/extra/python_tool.py
      severity: 2
      syntactic_context: 'wrapped,

        shell=True,

        cwd=str(self.base_path),

        encoding="utf8",

        executable="/bin/bash",

        env=env,

        stdout=subprocess.PIPE,

        stderr=subprocess.PIPE,

        )

        stdout, stderr = v.communicate()

        after = time()'
      tool_id: r2c.bandit
    30fad05838897b7ff4f15ffc8e194660:
      check_id: B602
      message: subprocess call with shell=True identified, security issue.
      path: tests/extra/test_python_tool.py
      severity: 2
      syntactic_context: 'proc = subprocess.Popen(

        cmd, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding="utf8"

        )

        stdout, stderr = proc.communicate()'
      tool_id: r2c.bandit
    5a4c9f7287dcc63f65ede98ac81ee151:
      check_id: B602
      message: subprocess call with shell=True identified, security issue.
      path: tests/extra/test_python_tool.py
      severity: 2
      syntactic_context: 'proc = subprocess.Popen(

        cmd, env=env, shell=True, stdout=subprocess.PIPE, encoding="utf8"

        )

        res, _ = proc.communicate()'
      tool_id: r2c.bandit
    c0441bba86cc9ce3775cbadde6f3fe06:
      check_id: B602
      message: subprocess call with shell=True identified, security issue.
      path: bento/extra/hadolint.py
      severity: 2
      syntactic_context: 'f"docker run --network none --rm -i {self.DOCKER_IMAGE}
        hadolint --format json - < {file}",

        shell=True,

        cwd=self.base_path,

        encoding="utf8",

        stdout=subprocess.PIPE,

        stderr=subprocess.PIPE,

        )

        hadolint_output = res.stdout

        hadolint_json = json.loads(hadolint_output)'
      tool_id: r2c.bandit
    cb09ff7333a5a433b189c697bfa69116:
      check_id: B602
      message: subprocess call with shell=True identified, security issue.
      path: tests/extra/test_python_tool.py
      severity: 2
      syntactic_context: 'env=env,

        shell=True,

        stdout=subprocess.PIPE,

        stderr=subprocess.PIPE,

        encoding="utf8",

        )

        stdout, stderr = proc.communicate()

        assert not stdout'
      tool_id: r2c.bandit
    f8ea89cbc1237d72dcb7ac244658cee3:
      check_id: B602
      message: subprocess call with shell=True identified, security issue.
      path: tests/extra/test_python_tool.py
      severity: 2
      syntactic_context: '# NOTE: This is a deliberate demonstration of a shell injection

        proc = subprocess.Popen(cmd, shell=True, stdout=subprocess.PIPE, encoding="utf8")

        res, _ = proc.communicate()'
      tool_id: r2c.bandit
r2c.eslint:
  violations:
    c6e0f687a580ef5e681dc3b4a7ed7fce:
      check_id: no-undef
      message: '''x'' is not defined.'
      path: tests/acceptance/bad.js
      severity: 2
      syntactic_context: x = "potato";
      tool_id: r2c.eslint
r2c.flake8:
  violations: {}
r2c.requests:
  violations: {}
